### ISSUE TITLE
sendrawtransaction response code -25, no message

### ISSUE THREAD

**Issue by rhaps107** [zcash/zcash#3152]
Hi, im trying to send v3 transaction on testnet, using node methods createrawtransaction, signrawtransaction, sendrawtransaction with transparent addresses.

1. create rawtransaction
request:
{"method":"createrawtransaction","params":[[{"vout":1,"txid":"318c608d7094d1d2fecb9856755885c9bc9027e8d0b8d342860007da2f2ea681","scriptPubKey":"76a914d02ff498b55babb5f649b3287c335e3c73d631d588ac","amount":2.99249997}],{"tmDykSoX2hQk5tseGaw4F3WgtqsHwsKPie3":0.00100001,"tmNhPLjb4Y6LxYbhEAS52t8Wsrfh9pvGxfN":0.00100001,"tmUh9L5M9koQWCEfMbH2q7sFfoLiZZ6QAbe":2.97549995}]}

response:
{"result":"030000807082c4030181a62e2fda07008642d3b8d0e82790bcc98558755698cbfed2d194708d608c310100000000ffffffff03a1860100000000001976a9142eca09954ca4411f6f30347b397e33d746adefee88aca1860100000000001976a9148e6ae59827a01e3ff07fa21748d1ee52c06da08588acab40bc11000000001976a914d02ff498b55babb5f649b3287c335e3c73d631d588ac00000000a14c030000","error":null,"id":null}

2. signrawtransaction
request: 
{"method":"signrawtransaction","params":["030000807082c4030181a62e2fda07008642d3b8d0e82790bcc98558755698cbfed2d194708d608c310100000000ffffffff03a1860100000000001976a9142eca09954ca4411f6f30347b397e33d746adefee88aca1860100000000001976a9148e6ae59827a01e3ff07fa21748d1ee52c06da08588acab40bc11000000001976a914d02ff498b55babb5f649b3287c335e3c73d631d588ac00000000a14c030000",[{"vout":1,"txid":"318c608d7094d1d2fecb9856755885c9bc9027e8d0b8d342860007da2f2ea681","scriptPubKey":"76a914d02ff498b55babb5f649b3287c335e3c73d631d588ac","amount":2.99249997}],["cVvKuJHNqW126Ha98TeoWavsBh1PuckgkA9moRVGaHzK9gjZgsxj"]]}

response: 
{"result":{"hex":"030000807082c4030181a62e2fda07008642d3b8d0e82790bcc98558755698cbfed2d194708d608c31010000006a4730440220602975d9c404299577b94a9da39589ec067910963bf260fbc744ea84908db3c60220072513ba54e9309c59c6ea4f57bcee8acb40d0e6e1a2d1e12496b5f384eb90da012103d6d0db608d40693996e6dd87b28c10ec90fd3b3c3d9c44e470a81f2cc1ce2201ffffffff03a1860100000000001976a9142eca09954ca4411f6f30347b397e33d746adefee88aca1860100000000001976a9148e6ae59827a01e3ff07fa21748d1ee52c06da08588acab40bc11000000001976a914d02ff498b55babb5f649b3287c335e3c73d631d588ac00000000a14c030000","complete":true},"error":null,"id":null}

3. sendrawtrasaction
request:
{"method":"sendrawtransaction", "params":["030000807082c4030181a62e2fda07008642d3b8d0e82790bcc98558755698cbfed2d194708d608c31010000006a4730440220602975d9c404299577b94a9da39589ec067910963bf260fbc744ea84908db3c60220072513ba54e9309c59c6ea4f57bcee8acb40d0e6e1a2d1e12496b5f384eb90da012103d6d0db608d40693996e6dd87b28c10ec90fd3b3c3d9c44e470a81f2cc1ce2201ffffffff03a1860100000000001976a9142eca09954ca4411f6f30347b397e33d746adefee88aca1860100000000001976a9148e6ae59827a01e3ff07fa21748d1ee52c06da08588acab40bc11000000001976a914d02ff498b55babb5f649b3287c335e3c73d631d588ac00000000a14c030000"]}

response:
{"result":null,"error":{"code":-25,"message":""},"id":null}

I got error with no message.

This transaction has 2 recepient addresses and 1 change address (you can all transaction creation details below).
If i make transaction with 1 recepient and 1 change same way, it works fine.

**Comment by bitcartel** [comment]
Using v1.1.0-rc1, I tried sending your transaction to testnet.

I see an error in the terminal window:
```
error code: -26
error message:
16: tx-overwinter-expired
```

I also see an error in debug.log:
```
2018-04-10 21:29:46 ERROR: ContextualCheckTransaction(): transaction is expired
2018-04-10 21:29:46 ERROR: AcceptToMemoryPool: ContextualCheckTransaction failed
```

That means the transaction has already expired, so you need to make it again and submit it within the (default) expiry time of 20 blocks.

Decoding the raw transaction confirms this, where the current block height is 218739:
```
  "txid": "d8227c6d38ec111bd2c991511f3fe7b6cf077c96c36459494145efaeee5f7edf",
  "overwintered": true,
  "version": 3,
  "versiongroupid": "03c48270",
  "locktime": 0,
  "expiryheight": 216225,
```

@rhaps107 Can you please try again with v1.1.0-rc? Thanks.

**Comment by rhaps107** [comment]
@bitcartel yes, transaction expired, thats why i posted methods to create trasaction. You can make 3 requests i mention to create new.

**Comment by ioptio** [comment]
Are you still having problems with new non-expiring transactions? Are you able to recreate this multiple times?
